Rubberduck.Core/AutoComplete/AutoCompleteService.cs

```diff
@@ -157,7 +157,8 @@ private bool DeleteAroundCaret(AutoCompleteEventArgs e, IAutoComplete autoComple
                 {
                     var left = code.Substring(0, e.CurrentSelection.StartColumn - 2);
                     var right = code.Substring(e.CurrentSelection.StartColumn);
-                    using (var pane = e.CodeModule.CodePane)
+                    using (var module = e.CodeModule)
```

Comment:
The event is raised [from here](https://github.com/rubberduck-vba/Rubberduck/blob/next/Rubberduck.VBEEditor/Events/VBENativeServices.cs#L202), and the wrapper is disposed by the event source.

Reply:
The main problem is that a SCW marshaled across an event arg is problematic because we don't know if the consumers will still need to reference the SCW at a later point outside its event handler. That would prevent the source from being able to dispose the SCW after it has fired its event. 

The cleanest method would be to marshal a QMN or equivalent. That way, there is no uncertainty on who owns the SCW, and will work in all cases regardless of whether the consumers only needs the reference within its handler or longer than that. 

Need an analyzer for that. 😆